### PR TITLE
Minor improvements on the explorer app

### PR DESCRIPTION
**Pull request by ghost**
Rearranged dimensions on the axes for a better viewing experience.
Added "Both" option to Compare by.
Added sorting option
Renamed coordinate categories to hint use case

### CODE REVIEW

File: sales2.csv [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Region,Product,Sales,Revenue [$]
```
Comment (blackary) [inline sales2.csv:1]: Did you mean to change the code to read from this file, rather than github raw?

File: example_app1.py [hunk 2]
```diff
@@ -5,10 +5,7 @@
 
 from streamlit_vizzu import VizzuChart
 
-data_frame = pd.read_csv(
-    "https://raw.githubusercontent.com/vizzuhq/ipyvizzu/main/docs/examples/stories/sales/sales.csv",
-    dtype={"tenure": str},
-)
+data_frame = pd.read_csv("https://raw.githubusercontent.com/vizzu-streamlit/streamlit-vizzu/main/sales2.csv")
```
Comment (blackary) [inline example_app1.py:8]:
```suggestion
data_frame = pd.read_csv("sales2.csv")
```

File: example_app1.py [hunk 3]
```diff
@@ -24,25 +21,30 @@
...
 )
 
-col1, col2, col3 = st.columns(3)
+col1, col2, col3, col4 = st.columns(4)
 
 measure: str = col1.radio("Measure", ["Sales", "Revenue [$]"])  # type: ignore
-compare_by = col2.radio("Campare by", ["Region", "Product"])
-coords = col3.radio("Coordinate system", ["Cartesian", "Polar"])
+compare_by = col2.radio("Compare by", ["Region", "Product", "Both"])
+coords = col3.radio("Coordinate system", ["Cartesian (desktop)", "Polar (mobile)"])
```
Comment (blackary) [inline example_app1.py:28]: What is the desktop/mobile for? Are they not supposed to work well on both platforms?

**Review by blackary — APPROVED** [review]
~~I don't understand the desktop/mobile parentheses, but otherwise~~ A couple suggestions, but otherwise this looks great.

File: example_app1.py (hunk 3 above)
Comment (blackary) [reply example_app1.py:28]: Nevermind, I see you noted that in the PR message. Maybe, if that's the suggested use-case, you could say "best for desktop/best for mobile" to make it more clear what they mean.

File: sales2.csv (hunk 1 above)
Comment (vizzu-streamlit) [reply sales2.csv:1]: No, I was just experimenting with the number formats - I don't like that some of the numbers don't fit into the markers, and I had to switch to another source file to do that.

**Pull request merged**